contract/src/test/resources/responses/invokescript_exception.json

```diff
@@ -0,0 +1,11 @@
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": {
+    "script": "DA5PcmFjbGVDb250cmFjdEEa93tn",
+    "state": "FAULT",
+    "gasconsumed": "0.0099915",
```

Comment:
Can you check other .json files for the tests? I think that now the `gasconsumed` attribute doesn't return decimal anymore. Please, adapt.

Also, make sure that in your tests you check if the correct amount of fees/gas added. This helps us, since if something changes, then this test fails. 😄

Reply:
I updated several tests now with the newest responses.